game.js

```diff
@@ -0,0 +1,62 @@
+const Frame = require("./frame");
```

Comment:
would have been good to see some guarding against edge cases, e.g. in the game if a player keeps entering in their score...